build1.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+#packages required to edit
+sudo apt-get install -qq squashfs-tools genisoimage
+#downloading the ISO to edit
+wget -q https://github.com/fossasia/meilix/releases/download/untagged-c9f68d2fa6b3d1cdbd99/meilix-zesty-20170611-i386.iso
+#exit on any error
+set -e
+
+mkdir mnt
+#Mount the ISO 
+sudo mount -o loop meilix-zesty-20170611-i386.iso mnt/
+#Extract .iso contents into dir 'extract-cd' 
+mkdir extract-cd
+sudo rsync --exclude=/casper/filesystem.squashfs -a mnt/ extract-cd
+#Extract the SquashFS filesystem 
+sudo unsquashfs -n mnt/casper/filesystem.squashfs
+sudo mv squashfs-root edit
+
+#test value of env variable
+echo $TRAVIS_SCRIPT
+
+export $TRAVIS_SCRIPT
+sudo su <<EOF
+echo "$TRAVIS_SCRIPT" > edit/meilix-generator.sh
+EOF
+
+#prepare chroot
+sudo mount -o bind /run/ edit/run
+sudo cp /etc/hosts edit/etc/
+sudo mount --bind /dev/ edit/dev
+
+sudo chroot edit <<EOF
+
+# execute environment variable
+ls # to test the files if any new file is added
+chmod +x meilix-generator.sh
+echo "$(<meilix-generator.sh)" #to test the file
+./meilix-generator.sh
```

Comment:
the script is to be generated by https://github.com/fossasia/meilix-generator/pull/76